unlonely-alpha/server/src/entities/User/userResolvers.ts

```diff
@@ -3,8 +3,10 @@ import { AuthenticationError } from "apollo-server-express";
 import { Context } from "../../context";
 import { authMessage } from "../../utils/auth";
 import * as userService from "./userService";
+import { JSONResolver } from "graphql-scalars";
 
 export const resolvers = {
+  JSON: JSONResolver,
```

Comment:
whats happening here?

Reply:
this is added to support json mapping for userChannel1155Mapping